common/supplemental/supplementalData.xml

```diff
@@ -4928,7 +4928,7 @@ XXX Code for transations where no currency is involved
 		<weekOfPreference ordering="weekOfYear weekOfInterval" locales="zu"/>
 		<weekOfPreference ordering="weekOfDate" locales="ca es fr gl"/>
 		<weekOfPreference ordering="weekOfDate weekOfMonth" locales="en bn ja ka"/>
-		<weekOfPreference ordering="weekOfDate weekOfMonth weekOfInterval" locales="bg de iw pt ur zh"/>
```

Comment:
there was a bug on this somewhere.

Reply:
Yes, there are some odd locations for 'iw'. The @MATCH cases now have these explicit variant locale tests, so we can review them.